src/SFA.DAS.Commitments.Infrastructure/Data/CommitmentRepository.cs

```diff
@@ -98,6 +98,7 @@ public CommitmentRepository(string databaseConnectionString, ICommitmentsLogger
                 parameters.Add("@commitmentStatus", commitment.CommitmentStatus, DbType.Int16);
                 parameters.Add("@editStatus", commitment.EditStatus, DbType.Int16);
                 parameters.Add("@lastAction", commitment.LastAction, DbType.Int16);
+                parameters.Add("@transferApprovalStatus", commitment.TransferApprovalStatus, DbType.Int16);
```

Comment:
Should we be changing the commitmentStatus in this way? Shouldn't it only get updated by the  ResetTransferApprovalStatus and the Sender Approving or rejecting.